migrations/m170914_110639_alter_roles_permissions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
```

Comment:
Can status and _status be made constants also?

Reply:
I should since they're column names, in order to increase readability

Reply:
Okay